src/api/generate-csv/map-application-to-csv/helpers/map-yes-no-field/index.ts

```diff
@@ -0,0 +1,21 @@
+import { DEFAULT } from '../../../../content-strings';
+
+/**
+ * mapYesNoField
+ * maps true or false fields to Yes/No
+ * @param {Boolean} answer
+ * @returns {String} Yes/No or original provided answer
+ */
+const mapYesNoField = (answer?: boolean) => {
+  if (answer === false) {
+    return 'No';
+  }
+
+  if (answer === true) {
+    return 'Yes';
```

Comment:
Constant please.

Reply:
Same as above for last 2 comments, can we do this after this PR as a tech improvement ticket, there's a few files that has yes/no without constants. We do have a constant for these just they're not being used in some instances

Reply:
Tickets created for both of these things: 
- https://ukef-dtfs.atlassian.net/browse/EMS-1421
- https://ukef-dtfs.atlassian.net/browse/EMS-1422